### ISSUE TITLE
Import suggestions and individual Lodash modules

### ISSUE THREAD

**Issue by mjbvz** [microsoft/TypeScript#25236]
_From @OliverJAsh on June 25, 2018 19:5_

Lodash provides individual modules for each of its functions, e.g.

``` ts
import maxBy from 'lodash/maxBy';
```

However, currently TypeScript/VSCode will not use these files for import suggestions:

![image](https://user-images.githubusercontent.com/921609/41870091-f0f1a634-78b2-11e8-8db7-b4f183b731d8.png)

… which creates

``` ts
import { maxBy } from "lodash";

maxBy
```

What would it take for TypeScript/VSCode to suggest these files?

I'm sure this pattern applies to other modules as well, not just Lodash.

_Copied from original issue: Microsoft/vscode#52831_

**Comment by RyanCavanaugh** [comment]
For many modules, you're not intended to reach into their subpaths to pull out individual parts. It feels like we should need some sort of hint that this is OK before we start encouraging you to depend on a random module's implementation details.

**Comment by OliverJAsh** [comment]
Alternatively, Lodash could bundle all module declarations into the index. This is what Recompose does, for example I get import suggestions for `pure` from `recompose/pure`.

**Comment by RyanCavanaugh** [comment]
There's a duplicate of this out there somewhere.
